gulpfile.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+/* eslint-env node */
+
+'use strict';
+
+const { exec } = require('child_process');
+
+const { parallel, watch } = require('gulp');
+
+function build(cb) {
+  exec('make build', (error, stdout, stderr) => {
+    console.error(stderr);
+    console.log(stdout);
+    cb(error);
+  });
```

Comment:
`make build` can take a few seconds to run and currently the output is only shown at the end. A possibility to improve this is to use `spawn` with `{ stdio: 'inherit' }` so that output from the command is forwarded as it is emitted by make:

```js
function build(cb) {
  const make = spawn('make', ['build'], { stdio: 'inherit' });
  make.on('close', code => {
    if (code !== 0) {
      cb(new Error(`make exited with status ${code}`));
    } else {
      cb(null);
    }
  });
}
```

Reply:
Great suggestion!